.vscode/settings.json

```diff
@@ -2,11 +2,15 @@
   "cSpell.words": [
     "AGRCGLOBAL",
     "autoplay",
+    "BASEMERIDIAN",
     "citytown",
     "firebaserc",
     "fullscreen",
     "hostingchannels",
     "nosniff",
-    "SAMEORIGIN"
+    "plss",
+    "SAMEORIGIN",
+    "SNUM",
+    "Ssearch"
```

Comment:
**question**: is this a typo?

Reply:
Probably part of TRSsearch. This project has that widget.